assets/queries/k8s/RBAC_wildcard/query.rego

```diff
@@ -0,0 +1,105 @@
+package Cx
+
+CxPolicy [result]  {
+	document := input.document[i]
+    metadata := document.metadata
+    document.kind == "ClusterRole"
```

Comment:
We could check the document.kind == ["Role", "ClusterRole"] and could delete the last 3 rules

Reply:
I've changed for functions